src/color/setSaturation.js

```diff
@@ -28,8 +28,9 @@ import curry from '../internalHelpers/_curry'
  * }
  */
 function setSaturation(saturation: number, color: string): string {
+  const hslColor = parseToHsl(color)
   return toColorString({
-    ...parseToHsl(color),
+    ...hslColor,
```

Comment:
ok I originally did this because we were doing it two different ways across color modules, I'll have to change this back as well as the update the ones where we were doing an assignment.